README.md

```diff
@@ -1,24 +1,13 @@
-# README
+# Todo app
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
 
-Things you may want to cover:
+* Ruby version:  "3.1.0"
```

Comment:
I would expect these to be headings rather than a bulleted list ie: "Dependencies", "Running the App", "Running the Tests" etc (group and name how you thing appropriate)

Reply:
done